### DISCUSSION TITLE
@quasar/vite-plugin-v1.0.0

### DISCUSSION THREAD

**Discussion by rstoenescu** [quasarframework/quasar#11101] [Announcements]
Here we go. Use the awesome Vite with Quasar through our Vite plugin.
https://quasar.dev/start/vite-plugin

**EDIT 1**: v1.0.1 has been released! Includes the first PR from the community (in less than a few hours after our first release), which is fantastic. Enjoy! https://github.com/quasarframework/quasar/releases/tag/%40quasar%2Fvite-plugin-v1.0.1

**EDIT 2**: Lifted the RTL support limitation for the Vite plugin. Information on how to add RTL support has been added to the docs.
https://quasar.dev/options/rtl-support (wait for docs PWA to pop up the "Refresh" request banner)

### Donations
...are very much appreciated as they help us keep going. https://donate.quasar.dev

<hr /><em>This discussion was created from the release <a href='https://github.com/quasarframework/quasar/releases/tag/%40quasar/vite-plugin-v1.0.0'>@quasar/vite-plugin-v1.0.0</a>.</em>

**Comment by hawkeye64** [comment]
Amazing! Thanks for your support to the community @rstoenescu !

**Reply by hasanerken** [reply]
It is a great news. Thanks for such a wonderful framework with Vite

**Comment by rstoenescu** [comment]
And `@quasar/vite-plugin` v1.0.1 has been released! Includes the first PR from the community (in less than a few hours after our first release), which is fantastic. Enjoy! https://github.com/quasarframework/quasar/releases/tag/%40quasar%2Fvite-plugin-v1.0.1

**Comment by rstoenescu** [comment]
Also lifted the RTL support limitation for the Vite plugin.
Information on how to add RTL support has been added to the docs.
https://quasar.dev/options/rtl-support (wait for docs PWA to pop up the "Refresh" request banner)
